Global UI Refactor

dev-charles15531 opened the pull request:
This PR contains the codes for #11, #12, & #14.
The lines commented in some store files should be left as seen, those are calls to my docker/local server, and i'll clean up when i'm done.

The calendar widget at top is a plugin. This plugin provides no way to customize the widget display style. I checked the style file, and figured it'll take a lot of time to modify to taste. so what do you think?

paradigmventures commented:
Amazing work on this PR!  Starting to take a look at it.

Formatting the dropdown calendar widget is not a top priority at the moment.  Perhaps later on in the nice-to-haves.

dev-charles15531 commented:
Thanks alot @paradigmventures ☺️

dev-charles15531 commented:
Hello @paradigmventures ,
i have added more fix to this PR. 

If you noticed, grid boxes tend to adgust in height and displace sibling grid boxes when events gets added to them(2+). I have modified the calendar layout to use flexbox and not grid(as provided by Tailwind UI). I have also limited the max number of events per gridbox to 2(previously 3) followed by the "+ x more" text.
The calendar looks more satisfying now no matter the number of events added to any day.
